site/en/configure/build-performance-breakdown.md

```diff
@@ -0,0 +1,233 @@
+Project: /_project.yaml
+Book: /_book.yaml
+
+# Breaking down build performance
+
+{% include "_buttons.html" %}
+
+Bazel is complex and does a lot of different things over the course of a build,
+some of which can have an impact on build performance. This page attempts to map
+some of these Bazel concepts to their implications on build performance. While
+not extensive, we have included some examples of how to detect build performance
+issues through [extracting metrics](/configure/build-performance-metrics)
+and what you can do to fix them. With this, we hope you can apply these concepts
+when investigating build performance regressions.
+
+### Clean vs Incremental builds
+
+A clean build is one that builds everything from scratch, while an incremental
+build reuses some already completed work.
+
+We suggest looking at clean and incremental builds separately, especially when
+you are collecting / aggregating metrics that are dependent on the state of
+Bazel’s caches (for example
+[build request size metrics](#deterministic-build-metrics-as-a-proxy-for-build-performance)
```

Comment:
use `_` instead of `-` for this section reference

Reply:
Done.